Sources/StarWarsAPI/StarWarsAPI/Sources/Fragments/CharacterNameWithInlineFragment.graphql.swift

```diff
@@ -63,8 +63,6 @@ public struct CharacterNameWithInlineFragment: StarWarsAPI.SelectionSet, Fragmen
 
       /// The movies this character appears in
       public var appearsIn: [GraphQLEnum<StarWarsAPI.Episode>?] { __data["appearsIn"] }
-      /// The name of the character
```

Comment:
This is an odd field to be removing; is it just catching up on generated code changes from prior PRs or is it relevant to this PR?